app/models/application_setting_format.rb

```diff
@@ -0,0 +1,63 @@
+class ApplicationSettingFormat < ApplicationRecord
+  store_accessor :settings
```

Comment:
What's the point in using `store_accessor` here?

Reply:
A leftover 😀 Thanks for pointing it out